doc/content.tex

```diff
@@ -1,1349 +0,0 @@
-                    Na\"{i}ve & 104.74 & 8.64 & \\
-                    Drift & 104.60 & 8.47 & \\
-                    Average & 79.91 & 7.14 & \\ \hline
-                    Simple exp. smoothing & 53.54 & 5.91 & 380.88 \\
-                    Double exp. smoothing & 44.41 & 5.28 & 375.90 \\
-                    Triple exp. smoothing & 5.34 & 1.71 & 282.73 \\ \hline
-                    Linear regression & 41.85 & 5.10 & 321.46 \\ \hline
-                    ARIMA(2, 1, 2) & 30.88 & 4.35 & 310.6 \\
-                    SARIMA(1, 0, 0)(1, 1, 0)[4] & 4.79 & 1.63 & 206.95 \\
-            \end{tabular}
-            \caption{Statistics of models.}
-            \label{tab:models-stat}
-        \end{center}
-    \end{table}
-
-    Statistics produced by each model are listed in table \ref{tab:models-stat}. Models with the lowest one
-    step ahead prediction error are seasonal ARIMA and triple exponential smoothing which is correct for this highly
-    seasonal time series.
-
-    As it was discussed in section \ref{sec:model-quality} AIC criterion should be used for models comparison. From the
-    table \ref{tab:models-stat} it can be seen that AIC and MSE are related, however MSE of double exponential smoothing
-    is $8.3$ times higher than MSE of triple exponential smoothing, whereas AIC of double exponential smoothing is
-    only $1.3$ times higher. From this example it is obvious how AIC discriminates models with more parameters.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Existing Solutions}
-Before going directly to the design and implementation let's briefly look at existing monitoring solutions and libraries
-for time series modelling.
-
-In the next section not only direct competitors of Hawkular are described but also
-generally available software of this kind. Because Hawkular is still only an upstream project and not deliverable
-product for the customers, thereby Jboss Operations Network (JON) is considered as its alternative.
-
-However, this work does not focus on precise comparison of monitoring applications. It is only mentioned for wider
-overview of this domain.
-
-    %%%%%%%%
-    \section{Monitoring and Management Applications}
-    Out there are many monitoring solutions available. Some of them are offered as service (SaaS) and other need to be
-    deployed in customer's environment.
-
-    For each application are highlighted the most important features and how it differs from its rivals. It is also
-    mentioned if the application offers predictive engine for metric data.
-
-        %%%%%%%%
-        \subsection{New Relic}
-        New Relic was the first monitoring solution offered as SaaS \cite{new-relic}. It focuses on application
-        performance rather than infrastructure view.
-
-        Monitoring of Java applications is done by an agents. Because agent manipulates with bytecode it is able to
```

Comment:
Because agent manipulates with bytecode it is able to 
? it v tej casti vety sa vztahuje v Tvojom ponimani na koho/co?